src/applications/disability-benefits/all-claims/config/781/index.js

```diff
@@ -105,6 +111,22 @@ export function createFormConfig781a(iterations) {
         uiSchema: secondaryIncidentDate.uiSchema(index),
         schema: secondaryIncidentDate.schema(index),
       },
+      [`secondaryUploadSourcesChoice${index}`]: {
```

Comment:
Does this need a unit test? I see one for `secondaryUploadSources`

Reply:
added unit test